scripts/list_updatable_packages

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+from collections import namedtuple
+from pathlib import Path
+from typing import Generator, Iterable, Mapping, Optional
+
+from debian.changelog import Changelog
+from debian.debian_support import NativeVersion
+
+import requests
+
+SESSION = requests.Session()
+
+
+Package = namedtuple('Package', ['package_name', 'version', 'gem_name', 'directory'])
```

Comment:
Recently I've taken to using [dataclasses](https://docs.python.org/3/library/dataclasses.html) but that requires Python 3.7.

Reply:
What would be the benefit? (besides type hints)

Reply:
The syntax feels more natural and you can have defaults. For example:
```python
@dataclasses.dataclass
class Package:
    package_name: str
    count: int = 0
    mylist: list[int] = dataclasses.field(default_factory=list)
```

The [field()](https://docs.python.org/3/library/dataclasses.html#dataclasses.field) method can also determine if the field is used in the generated methods like `__repr__` and __eq__`.

I quite like it over `namedtuple`, especially if you want to add some additional methods which is always awkward with `namedtuple`.